### ISSUE TITLE
Revise global nav

### ISSUE THREAD

**Issue by kissane** [COVID19Tracking/website#365]
Now that we don't have the footer, we've lost some things and need to do some revision. Can someone document process for changing global nav rn? Is that hard-coded? If so, shall I just put changes here?

**Comment by kevee** [comment]
Just let us know and we can change it. Here's the docs:

https://covid19tracking.github.io/website-docs/?path=/docs/main-navigation--page

**Comment by kissane** [comment]
Wonderful. After the fires are all out today, let's revise to:

```
- title: Get the Data
    link: /data
- title: About the Data
    link: /about-data
- title: About the Project
    link: /about-project
- title: Data API
    link: /api
- title: How You Can Help
    link: /help
```

**Comment by kevee** [comment]
Fixed in #373
